composer.json

```diff
@@ -15,9 +15,9 @@
     "laminas/laminas-cache-storage-implementation": "1.0"
   },
   "require-dev": {
-    "laminas/laminas-cache": "3.0.x-dev",
-    "laminas/laminas-cache-storage-adapter-test": "2.0.x-dev",
-    "laminas/laminas-coding-standard": "~2.2.0",
+    "laminas/laminas-cache": "3.1.x",
+    "laminas/laminas-cache-storage-adapter-test": "2.2.x",
```

Comment:
Hmm, I think renovate is quite confused here 🤔

Reply:
It's respecting the constraint currently applied. Since this versions are pinned to `3.0.x-dev` and `2.0.x-dev` respectively, it's performing the upgrade whilst keeping them pinned to specific versions.

Reply:
Aware, but it dropped `-dev` :) 

Anyway, these are released now, and we can move to stable ranges.